docs/source/_templates/layout.html

```diff
@@ -0,0 +1,48 @@
+{% extends "!layout.html" %}
+
+{% block extrahead %}
+{{ super() }}
+{% endblock %}
+
+
+<!-- START OF LOCAL OVERRIDE -->
+<!--  This block overrides the theme to enable third level left navigation.-->
+{% block menu %}
+    {% if 'singlehtml' not in builder %}
+         {% set global_toc = toctree(collapse=theme_collapse_navigation|tobool,
+                                     includehidden=theme_includehidden|tobool,
+                                     titles_only=theme_titles_only|tobool) %}
+         {% endif %}
+         {% if global_toc %}
+              {{ global_toc }}
+         {% else %}
+              <!-- Local TOC -->
+              <div class="local-toc">{{ toc }}</div>
+         {% endif %}
+{% endblock %}
+<!-- END OF LOCAL OVERRIDE -->
+
+<script script type="text/javascript">
```

Comment:
It should match what you have in the toctree caption. It looks like you have only toctree captions at the moment:
'Introduction', 'All API References'. 

```suggestion
<script type="text/javascript">
```